eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json

```diff
@@ -0,0 +1,746 @@
+{
+  "process_graph": {
+    "1": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "31"
+        },
+        "cube1": {
+          "from_node": "10"
+        },
+        "overlap_resolver": {
```

Comment:
The empty overlap resolver is why the web editor complains, not sure why that got generated multiple times.

Reply:
see #115